### PR TITLE
Link association.exists?(...) to AR::Base.exists? guide [ci skip]

### PR DESCRIPTION
**Pull request by eliotsykes**

### CODE REVIEW

File: guides/source/association_basics.md [hunk 1]
```diff
@@ -1506,7 +1506,7 @@ The `collection.where` method finds objects within the collection based on the c
 
 ##### `collection.exists?(...)`
 
-The `collection.exists?` method checks whether an object meeting the supplied conditions exists in the collection. It uses the same syntax and options as `ActiveRecord::Base.exists?`.
+The `collection.exists?` method checks whether an object meeting the supplied conditions exists in the collection. It uses the same syntax and options as [`ActiveRecord::Base.exists?`](active_record_querying.html#existence-of-objects).
```
Comment (senny) [inline guides/source/association_basics.md]: Can you hard wrap the modified line? Also I'd rather link to the API docs instead.

File: guides/source/association_basics.md [hunk 2]
```diff
@@ -2004,7 +2004,7 @@ The `collection.where` method finds objects within the collection based on the c
...
```
Comment (senny) [inline guides/source/association_basics.md]: Same as https://github.com/rails/rails/pull/20255/files#r30874007

**Comment by eliotsykes** [comment]
Thanks @senny, wrapped at <= 80 chars - I don't know if thats the right number of chars

**Comment by senny** [comment]
@eliotsykes thank you :yellow_heart:

**Pull request merged**